test/functional/tools/identifier_source.xml

```diff
@@ -0,0 +1,19 @@
+<tool id="identifier_source" name="identifier_source" version="0.1.0">
+    <description>verifies identifier source can be specified from input</description>
+    <command>
+<![CDATA[
+    cat $inputA $inputB > outputA &&
+    cat $inputA $inputB > outputA
```

Comment:
Can you single-quote inputs and outputs?
Also, on this line I suppose `outputA` should be `$outputB`